module: localization
topics:
  # ipm语义分割结果
  ipm_seg: /ipm_seg
  # imu数据
  imu: /imu
  #gps
  gps_odom: /gps_odom

  #底盘:
  chassis: /chassis

  #发布定位数据
  out_pose: /out_pose

data:
  rosbag_path: /home/hucunwei/study/avp/assignment/L9/data/avp_localization.bag
  map_path: /home/hucunwei/study/avp/assignment/L9/data/avp_map.bin

#初始位置和航向（度）
# init_pos:
#   x: 0.0
#   y: 0.0
#   yaw: 10.0
  
